pyth_observer/crosschain.py

```diff
@@ -48,7 +48,8 @@ async def get_crosschain_prices(self) -> Dict[str, CrosschainPrice]:
                 async with session.get(
                     price_feeds_url + query_string,
                 ) as response:
-                    price_feeds += await response.json()
+                    response_json = await response.json()
+                    price_feeds.extend(response_json["parsed"])
```

Comment:
Not sure how this was working before, but this wouldn't run locally for me without this change because the json response structure wasn't as expected.

Reply:
@cctdaniel any idea on this one?

Reply:
hmm i can see that the `query_string` is changed in this PR https://github.com/pyth-network/pyth-observer/pull/79/files#diff-858c896a341166e34351e654992a17790df86095bb42d1b262b9673d43e2ea98R47 it seems like it wasnt tested properly

Reply:
prod is still using [v0.2.14](https://github.com/pyth-network/deployments/blob/main/environments/platform-yellow/observer-pythnet/deployment.yaml#L20) which is why its working but `observer-pythtest` is using v0.2.15 which i assume is probably broken

Reply:
I see, thanks -- I'll leave this change in then, and verify it works in testnet before deploying to mainnet